atc/worker/volume.go

```diff
@@ -30,6 +30,7 @@ type Volume interface {
 	COWStrategy() baggageclaim.COWStrategy
 
 	InitializeResourceCache(db.UsedResourceCache) error
+	InitializeStreamedResourceCache(db.UsedResourceCache, string) error
```

Comment:
If we remove the condition of `""` as being valid for `db.CreatedVolume.InitializeResourceCache`, then we can get rid of this as well

Reply:
As replied to the previous comment, I intentionally added it.